src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/style.ts

```diff
@@ -68,14 +68,23 @@ export const StyledNewTabLink = styled(NewTabLink)`
   }}
 `;
 
-const DROPDOWN_WIDTH = "200px";
+const DROPDOWN_WIDTH = "209px";
 
 // High specificity label styling is to beat out czifui coloring.
 export const StyledDropdown = styled(Dropdown)`
   width: ${DROPDOWN_WIDTH};
   .MuiButton-label > span {
     color: black;
   }
+
+  ${(props) => {
+    const colors = getColors(props);
+    return `
+      path {
+        fill: ${colors?.gray[500]};
```

Comment:
I thought the coloring aspect was going to get fixed upstream in SDS instead? Will this lock the dropdown carat color to always being gray, or will it still alternate into primary color when open because of some CSS specificity thing I'm missing?

Reply:
this will be fixed, but this ticket opened up a whole can of worms about how sds design thinks about "active" state, so it might take longer than expected to get this fix in upstream